sdk/appconfiguration/dirs.proj

```diff
@@ -0,0 +1,5 @@
+<Project Sdk="Microsoft.Build.Traversal">
```

Comment:
nit: we usually use 2 space indention for xml based project files.

Reply:
Addressed.